DRIVERS/CDH_COMMON/src/Device_Startup/startup_samv71q21b.c

```diff
@@ -136,6 +137,10 @@ void GMAC_Q3_Handler      ( void ) __attribute__ ((weak, alias("Dummy_Handler"))
 void GMAC_Q4_Handler      ( void ) __attribute__ ((weak, alias("Dummy_Handler")));
 void GMAC_Q5_Handler      ( void ) __attribute__ ((weak, alias("Dummy_Handler")));
 
+// Suppress pedantic warnings as its technically UB to cast a fp to a void pointer, but this cant be fixed without
+//  editing the atmel drivers
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wpedantic"
```

Comment:
Not required for this PR but the warning spam was annoying me and theres nothing we can do about the warnings